contracts/utils/contracts/test/TestRefundable.sol

```diff
@@ -1,49 +0,0 @@
-/*
```

Comment:
ditto

Reply:
This was testing the `Refundable` contract which was removed in the same commit https://github.com/0xProject/protocol/pull/643/commits/d19e8eead276d8191c071f7f41c39df6d2f710f7